scripts/install

```diff
@@ -38,76 +42,76 @@ grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
 }
 
 # Install dependencies
-ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control \
-  || ynh_die "Unable to install dependencies"
+ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control
```

Comment:
Here you could also use the newer ynh_install_app_dependencies helper and avoid using a .control file.